File: DoneIt_Version_Movil/app/src/main/java/com/herreroco/doneit_version_movil/model/EditarTareaRequest.kt
package com.herreroco.doneit_version_movil.model

data class EditarTareaRequest(
    val titulo: String,
    val descripcion: String?,
    val fecha_Inicio: String?,
    val fecha_Fin: String?,
    val estado: String,
    val prioridad: String
)

File: DoneItAPI/DoneItAPI/DTOs/EditarTareaRequest.cs
namespace DoneItAPI.DTOs
{
    public class EditarTareaRequest
    {
        public string Titulo { get; set; }
        public string? Descripcion { get; set; }
        public DateTime? Fecha_Inicio { get; set; }
        public DateTime? Fecha_Fin { get; set; }
        public string Estado { get; set; }
        public string Prioridad { get; set; }
    }
}
